gulpfile.js

```diff
@@ -142,4 +159,38 @@ gulp.task('watch', function() {
 	gulp.watch('**/*.css', ['lint:css']);
 });
 
+// Server code from http://blog.overzealous.com/post/74121048393/why-you-shouldnt-create-a-gulp-plugin-or-how-to
+gulp.task('server', ['build:extensions'], function(callback) {
+	var log = gutil.log;
+	var colors = gutil.colors;
+
+	var devApp = connect();
+	devApp.use(connectLogger('dev'));
+	devApp.use(connectStatic('.'));
+
+	// Automatically rebuild Extensions on script changes
+	gulp.watch('**/*.js', ['build:extensions']);
```

Comment:
Do we need to watch `**/*` if we're only rebuilding the extensions?  Maybe a tighter `Extensions/**/*` here instead?